DNN Platform/Library/Services/Scheduling/DNNScheduler.cs

```diff
@@ -54,19 +54,13 @@ public class DNNScheduler : SchedulingProvider
 
         #region Private Methods
 
-        private bool CanRunOnThisServer(string servers)
+        private static bool CanRunOnThisServer(string servers)
         {
-            string lwrServers = "";
-            if (servers != null)
-            {
-                lwrServers = servers.ToLower();
-            }
-            if (String.IsNullOrEmpty(lwrServers) || lwrServers.Contains(Globals.ServerName.ToLower()))
-            {
-                return true;
-            }
-
-            return false;
+            return
+                // Scheduler should run on every server
+                string.IsNullOrWhiteSpace(servers)
+                // Check if scheduler should run on this server
+                || servers.Split(',').Contains(Globals.ServerName, StringComparer.InvariantCultureIgnoreCase);
```

Comment:
In general, `InvariantCulture` and `InvariantCultureIgnoreCase` aren't really useful.  For non-culture-specific comparisons, `Ordinal` and `OrdinalIgnoreCase` should be used (see https://msdn.microsoft.com/en-us/library/dd465121.aspx).